driver/src/main/scala/core/nodeset.scala

```diff
@@ -667,6 +655,53 @@ final class ChannelFactory private[reactivemongo] (
     pipeline
   }
 
+  private def sslContext = {
+    import java.io.FileInputStream
+    import java.security.KeyStore
+    import javax.net.ssl.{ KeyManagerFactory, TrustManager }
+
+    val keyManagers = Option(System.getProperty("javax.net.ssl.keyStore")).map { path =>
+
+      val password = Option(System.getProperty("javax.net.ssl.keyStorePassword")).getOrElse("")
```

Comment:
having to revert back to a static password as uuidgen not found
https://travis-ci.org/opetch/ReactiveMongo/jobs/185468041